File: lib_common/src/main/java/com/app/org/http2/IHttpService.java
package com.app.org.http2;

import com.app.org.http.BaseHttpModel;
import com.app.org.testhttp.IHttpListener;

/**
 * Created by lixingxing on 2018/5/10.
 */
public interface IHttpService {
    void request(BaseHttpModel baseHttpModel);


    void setUrl(String url);
    void setRequest(byte[] requestData);
    //进行两个接口的通信
    void setHttpCallBack(IHttpListener httpListener);
    void execute();
}

File: lib_common/src/main/java/com/app/org/http/interfaces/IHttpService.kt
package com.app.org.http.interfaces

import com.app.org.http.BaseHttpModel
import com.app.org.http.Response

/**
 * Created by lixingxing on 2018/5/10.
 */
abstract class IHttpService {
    var isShowLog = true
    var tag = "IHttpService"
    abstract fun request(baseHttpModel: BaseHttpModel): Response

}
